presto-native-execution/scripts/setup-macos.sh

```diff
@@ -17,51 +17,24 @@ set -eufx -o pipefail
 source "$(dirname "${BASH_SOURCE}")/../velox/scripts/setup-macos.sh"
 
 MACOS_DEPS="${MACOS_DEPS} bison gperf libsodium"
-export FB_OS_VERSION=v2022.11.14.00
+export FB_OS_VERSION=v2023.12.04.00
+
 
 function install_six {
   pip3 install six
 }
 
 export PATH=$(brew --prefix bison)/bin:$PATH
 
-function install_folly {
-  github_checkout facebook/folly "${FB_OS_VERSION}"
-  OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1) \
-   cmake_install -DBUILD_TESTS=OFF -DFOLLY_HAVE_INT128_T=ON
-}
-
-function install_fizz {
-  github_checkout facebookincubator/fizz "${FB_OS_VERSION}"
-  OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1) \
-    cmake_install -DBUILD_TESTS=OFF -S fizz
-}
-
-function install_wangle {
-  github_checkout facebook/wangle "${FB_OS_VERSION}"
-  OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1) \
-    cmake_install -DBUILD_TESTS=OFF -S wangle
-}
-
-function install_fbthrift {
-  github_checkout facebook/fbthrift "${FB_OS_VERSION}"
-  OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1) \
-    cmake_install -DBUILD_TESTS=OFF
-}
 
 function install_proxygen {
   github_checkout facebook/proxygen "${FB_OS_VERSION}"
-  OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1) \
-    cmake_install -DBUILD_TESTS=OFF
+  cmake_install -DBUILD_TESTS=OFF
 }
 
 function install_presto_deps {
   install_velox_deps
-  run_and_time install_folly
   run_and_time install_six
-  run_and_time install_fizz
```

Comment:
Why are we no longer installing fizz and others?

Reply:
Fizz and others have been moved to the Velox setup scripts.

Reply:
> Fizz and others have been moved to the Velox setup scripts.

I missed that. That explains why we started seeing fizz-related failures on PRs. Looks like we didn't communicate addition of these dependencies to Velox.

https://github.com/facebookincubator/velox/pull/8494?fbclid=IwAR19JnG6_rot86pZzqaZWxF-g60zP7juSEH7gwoW01n6vMqM6r9Y3-rl68g#issuecomment-1916815212

Reply:
This is the PR that updated the Velox side scripts https://github.com/facebookincubator/velox/pull/8454

Reply:
> That explains why we started seeing fizz-related failures on PRs. Looks like we didn't communicate addition of these dependencies to Velox.

This was not new for Velox. It was added here originally https://github.com/facebookincubator/velox/pull/5743/files
The failure could be due to the newer versions. I will take a look. Did not notice that issue.